### ISSUE TITLE
1.93.1: fatal error: 'avogadromolequeueexport.h' file not found

### ISSUE THREAD

**Issue by yurivict** [OpenChemistry/avogadrolibs#554]
**Avogadro version: (please complete the following information from the About box):**
 - Avogadrolibs: 1.93.1
 - Qt: 5.15.2

```
In file included from /disk-samsung/freebsd-ports/science/avogadrolibs/work/avogadrolibs-1.93.1/avogadro/qtplugins/cp2kinput/cp2kinputdialog.cpp:24:
/disk-samsung/freebsd-ports/science/avogadrolibs/work/avogadrolibs-1.93.1/avogadro/molequeue/molequeuedialog.h:20:10: fatal error: 'avogadromolequeueexport.h' file not found
#include "avogadromolequeueexport.h"
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~
1 warning and 1 error generated.
```

**Comment by ghutchis** [comment]
This is a duplicate of #527 . Are you compiling Molequeue? If not, it's a duplicate of #514 

That's not to say we aren't working to separate, but I don't understand why you need the duplicate issues.

**Comment by yurivict** [comment]
I have molequeue-0.9.0_3 installed.

**Comment by yurivict** [comment]
Where avogadromolequeueexport.h is supposed to come from?

**Comment by ghutchis** [comment]
It's generated in `avogadro/CMakeLists.txt`:
https://github.com/OpenChemistry/avogadrolibs/blob/master/avogadro/CMakeLists.txt

Do you have `-DBUILD_MOLEQUEUE=OFF`

**Comment by ghutchis** [comment]
This should be fixed by #558 - this is a duplicate of #514 - sorry that I didn't see the `cp2k` piece there.

**Comment by yurivict** [comment]
Commit in #558 didn't fix the issue.

I build with:
```
-DUSE_HDF5:BOOL=false -DUSE_LIBARCHIVE:BOOL=false -DUSE_LIBMSYM:BOOL=false -DUSE_MOLEQUEUE:BOOL=false -DUSE_OPENGL:BOOL=true -DUSE_QT:BOOL=true -DUSE_SPGLIB:BOOL=false -DUSE_VTK:BOOL=false -DCMAKE_C_COMPILER:STRING="cc"  -DCMAKE_CXX_COMPILER:STRING="c++"  -DCMAKE_C_FLAGS:STRING="-O2 -pipe -fno-omit-frame-pointer  -fstack-protector-strong -fno-strict-aliasing "  -DCMAKE_C_FLAGS_DEBUG:STRING="-O2 -pipe -fno-omit-frame-pointer  -fstack-protector-strong -fno-strict-aliasing "  -DCMAKE_C_FLAGS_RELEASE:STRING="-O2 -pipe -fno-omit-frame-pointer  -fstack-protector-strong -fno-strict-aliasing "  -DCMAKE_CXX_FLAGS:STRING="-O2 -pipe -fno-omit-frame-pointer -fstack-protector-strong -fno-strict-aliasing -fno-omit-frame-pointer  "  -DCMAKE_CXX_FLAGS_DEBUG:STRING="-O2 -pipe -fno-omit-frame-pointer -fstack-protector-strong -fno-strict-aliasing -fno-omit-frame-pointer  "  -DCMAKE_CXX_FLAGS_RELEASE:STRING="-O2 -pipe -fno-omit-frame-pointer -fstack-protector-strong -fno-strict-aliasing -fno-omit-frame-pointer  "  -DCMAKE_EXE_LINKER_FLAGS:STRING=" -fstack-protector-strong "  -DCMAKE_MODULE_LINKER_FLAGS:STRING=" -fstack-protector-strong "  -DCMAKE_SHARED_LINKER_FLAGS:STRING=" -fstack-protector-strong "  -DCMAKE_INSTALL_PREFIX:PATH="/usr/local"  -DCMAKE_BUILD_TYPE:STRING="Release"  -DTHREADS_HAVE_PTHREAD_ARG:BOOL=YES  -DCMAKE_INSTALL_RPATH_USE_LINK_PATH=YES  -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON -DBUILD_SHARED_LIBS:BOOL=ON -GNinja
```
